CMakeLists.txt

```diff
@@ -495,6 +495,76 @@ while(library_variants_copy)
             -DLIBCXX_ENABLE_WIDE_CHARACTERS=OFF
             -DLIBUNWIND_ENABLE_THREADS=OFF
         )
+
+        # picolibc tests
+        string(REPLACE " " ";" qemu_params "${qemu_params}")
+        add_custom_target(check-picolibc_${variant})
+        set(qemu_command "qemu-system-${cpu_family}")
+        set(picolibc_tests
+            abort
+            atexit
+            complex-funcs
+            constructor
+            constructor-skip
+            fenv
+            ffs
+            hosted-exit
+            long_double
+            malloc
+            malloc_stress
+            math_errhandling
+            math-funcs
+            on_exit
+            posix-io
+            printf_scanf
+            printf-tests
+            rand
+            regex
+            setjmp
+            stack-smash
+            test-efcvt
+            test-except
+            test-fopen
+            test-memset
+            test-mktemp
+            test-put
+            test-strchr
+            test-strtod
+            timegm
+            time-sprintf
+            time-tests
+            tls
+            ungetc
+        )
+
+        function (picolibc_test test)
```

Comment:
The linter doesn't like the space after `function`

Reply:
Fixed.